lib/SILOptimizer/LoopTransforms/LoopBranchHoist.cpp

```diff
@@ -0,0 +1,321 @@
+//===--- LoopBranchHoist.cpp - Hoist conditional branches out of loops ----===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#define DEBUG_TYPE "sil-loopbranchhoist"
+
+#include "llvm/ADT/DepthFirstIterator.h"
+
+#include "swift/SIL/PatternMatch.h"
+#include "swift/SIL/SILCloner.h"
+#include "swift/SILOptimizer/Analysis/LoopAnalysis.h"
+#include "swift/SILOptimizer/PassManager/Passes.h"
+#include "swift/SILOptimizer/PassManager/Transforms.h"
+#include "swift/SILOptimizer/Utils/PerformanceInlinerUtils.h"
+#include "swift/SILOptimizer/Utils/SILInliner.h"
+#include "swift/SILOptimizer/Utils/SILSSAUpdater.h"
+
+using namespace swift;
+using namespace swift::PatternMatch;
+
+using llvm::DenseMap;
```

Comment:
I think these are already in the swift namespace via the header swift/Basic/LLVM.h.